CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_LIST_DIR}/share/rocmcmakebuildtool
 include(ROCMCreatePackage)
 include(ROCMSetupVersion)
 
-rocm_setup_version(VERSION 0.11.0)
+rocm_setup_version(VERSION 0.12.0)
```

Comment:
Should this be 0.13.0?

Reply:
created https://github.com/ROCm/rocm-cmake/pull/210 to address this. will cherry pick that change into this PR if it is merged